### DISCUSSION TITLE
Display repeater options horizontally

### DISCUSSION THREAD

**Discussion by l-mtc** [filamentphp/filament#10006] [Help]
Is there a nice way to display these horizontally without impacting the child styling?

**Comment by leandrocfe** [comment]
What is the issue?

**Reply by l-mtc** [reply]
<img width="1069" alt="Screenshot 2023-12-01 at 20 03 38" src="https://github.com/filamentphp/filament/assets/146552524/b098f4f2-0704-4dd0-a50d-b10275f15b1d">

**Reply by l-mtc** [reply]
I want to have these options in 1 row, but not have the child elements take up a smaller portion of the grid

**Reply by leandrocfe** [reply]
I think you can't do it

**Comment by AFSDARIF** [comment]
using this https://filamentphp.com/docs/3.x/forms/fields/repeater#grid-layout

use Filament\Forms\Components\Repeater;
 
Repeater::make('qualifications')
    ->schema([
        // ...
    ])
    ->grid(2)

![image](https://github.com/user-attachments/assets/638116ce-36e9-40f8-bee0-214e6b6df186)
